src/components/Featured/projects.js

```diff
@@ -4,8 +4,8 @@ import gitParseImage from "../../images/featured/git-parse.png"
 
 const projects = [
   {
-    description: `<code>git-parse</code> allows JavaScript developers
-      to treat a git repository as they would treat a regular object. 
+    description: `<a href="https://github.com/wayfair/git-parse"><code>git-parse</code></a> allows JavaScript developers
+      to treat a git repository as they would treat a regular object.
```

Comment:
I like the simplicity of this. I'm curious if it might make more sense to attempt putting links as another part of this data structure so the _title_ of the project shown as part of the carousel is also clickable. What do you think?

Reply:
woops -- I thought we had titled with each of these... but I guess not :)